legacy_db/mysql2sqlite2

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/awk -f
+
```

Comment:
Add documentation to top of document saying what this is for (since you didn't write it)